contrib/codegen/templates/sets/sets.gotmpl

```diff
@@ -6,6 +6,7 @@ import (
     {{ $import_prefix }} "{{ group_import_path $ }}"
 
     "github.com/rotisserie/eris"
+    "github.com/solo-io/go-utils/contextutils"
```

Comment:
Why is this import needed?

Reply:
This was leftover from a previous change, fixed